app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -57,12 +64,20 @@ def authenticated_with_all_providers?
         identities.count == SiteConfig.authentication_providers.count
       end
 
+      def user_is_following_tags?
+        user.cached_followed_tag_names.count > 1
+      end
+
       def welcome_broadcast
         @welcome_broadcast ||= Broadcast.find_by(title: "Welcome Notification: welcome_thread")
       end
 
-      def customize_broadcast
-        @customize_broadcast ||= Broadcast.find_by(title: "Welcome Notification: customize_experience")
+      def customize_ux_broadcast
+        @customize_ux_broadcast ||= Broadcast.find_by(title: "Welcome Notification: customize_experience")
+      end
+
+      def customize_feed_broadcast
+        @customize_feed_broadcast ||= Broadcast.find_by(title: "Welcome Notification: customize_feed")
```

Comment:
What is the plan for creating this new Broadcast in production?

Reply:
@mstruve Currently, we are adding them via the `/internal/broadcasts` page as part of the deployment process. Notably, none of this stuff is live to users. But we are adding them as we merge.

Ultimately, however, we want to make these broadcasts more flexible and modifiable by admins for different communities! We are still working out, as a team, the way that we want to do that. Down the road, we likely will be using the data update scripts to pre-populate some generic Broadcasts for the welcome notification flow, and then allow admins of different communities to turn this feature off/on explicitly.

Reply:
Excellent, just wanted to make sure we were thinking about all of that stuff. Thanks for the great explanation!